A2rchi/utils/mailbox.py

```diff
@@ -1,9 +1,14 @@
 #!/bin/python
-import os,sys
-import getpass, imaplib, email
+from A2rchi.utils.config_loader import Config_Loader
+from A2rchi.utils.env import read_secret
+
+import email
+import imaplib
+import os
+
+# DEFINITIONS
+ISSUE_ID_OFFSET = 9
```

Comment:
What does this do?

Reply:
Good question! So on line 105 there is a statement that indexes into a string by 9 characters to get the issue id from a description. One general guideline for code cleanliness is to try to limit the number of integers floating around in the source code, especially when their semantic meaning may be hard to interpret. The common solution is to define these constants with a semantically meaningful name at the top of the file. 

Note that `0`, `-1`, and `1` are notable exceptions to this rule because they are commonly understood to be indexing the beginning and end of lists or correcting off-by-one issues. Also, in application specific scenarios it may be obvious why an integer is chosen, which eliminates the need for defining a constant (e.g. using `n % 2` in an `if` statement to filter for even numbers).

Reply:
This makes sense, we should document what you wrote above as a comment for this offset, to make it clear what it is. We can save this for a later PR though